src/jdk.jfr/share/classes/jdk/jfr/Throttle.java

```diff
@@ -24,53 +24,57 @@
  * questions.
  */
 
-package jdk.jfr.internal;
+package jdk.jfr;
 
 import java.lang.annotation.ElementType;
 import java.lang.annotation.Inherited;
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
-import jdk.jfr.MetadataDefinition;
-
 /**
- * Event annotation, determines the event emission rate in events per time unit.
- *
- * This setting is only supported for JVM events.
+ * Event annotation, specifies the maximum rate of events per time unit, (for
+ * example, {@code "100/s"}).
+ * <p>
+ * If the event class annotated with {@code Throttle} is filtered by other
+ * settings, such as a {@link jdk.jfr.Threshold} or a user-defined setting, the
+ * throttling will happen after those settings have been applied.
  *
- * @since 16
+ * @since 25
  */
 @MetadataDefinition
 @Target({ ElementType.TYPE })
 @Inherited
 @Retention(RetentionPolicy.RUNTIME)
 public @interface Throttle {
     /**
-     * Settings name {@code "throttle"} for configuring an event emission rate in events per time unit.
+     * Setting name {@code "throttle"} for configuring throttled events.
      */
     public static final String NAME = "throttle";
-    public static final String DEFAULT = "off";
 
     /**
-     * Throttle, for example {@code "100/s"}.
+     * The throttle rate, for example {@code "100/s"}.
      * <p>
-     * String representation of a non-negative {@code Long} value followed by a slash ("/")
-     * and one of the following units<br>
-     * {@code "ns"} (nanoseconds)<br>
-     * {@code "us"} (microseconds)<br>
-     * {@code "ms"} (milliseconds)<br>
-     * {@code "s"} (seconds)<br>
-     * {@code "m"} (minutes)<br>
-     * {@code "h"} (hours)<br>
-     * {@code "d"} (days)<br>
+     * String representation of a non-negative {@code long} value followed by a
+     * forward slash ("/") and one of the following units: <br>
+     * <ul style="list-style-type:none">
+     * <li>{@code "ns"} (nanoseconds)</li>
+     * <li>{@code "us"} (microseconds)</li>
+     * <li>{@code "ms"} (milliseconds)</li>
+     * <li>{@code "s"} (seconds)</li>
+     * <li>{@code "m"} (minutes)</li>
+     * <li>{@code "h"} (hours)</li>
+     * <li>{@code "d"} (days)</li>
+     * </ul>
      * <p>
      * Example values, {@code "6000/m"}, {@code "10/ms"} and {@code "200/s"}.
-     * When zero is specified, for example {@code "0/s"}, no events are emitted.
-     * When {@code "off"} is specified, all events are emitted.
+     * <p>
+     * Specifying zero, for example {@code "0/s"}, results in no events being
+     * emitted.
+     * <p>
+     * Specifying {@code "off"} (case-sensitive) results in all events being emitted.
      *
      * @return the throttle value, default {@code "off"} not {@code null}
```

Comment:
I think it would be clear to drop "not null" from the return description.

Reply:
We have "not null" with Threshold, but not with "Period". I filed a bug to make it consistent for all JFR annotations and I can bring Throttle into the same CSR. 

https://bugs.openjdk.org/browse/JDK-8358602